packages/dx-grid-core/src/plugins/sorting-state/helpers.js

```diff
@@ -1,4 +1,25 @@
+const mergeWithUnique = (arr1, arr2) => [...new Set([...arr1, ...arr2])];
```

Comment:
Can we split this function to merge and unique?

Reply:
Hm, it's a single-line function... Yes, we can split it, but I'm not sure we have to.

Reply:
It is not good practice to do two separate actions in a util function